### ISSUE TITLE
The turning capability does not correspond to the Steering angle.

### ISSUE THREAD

**Issue by malamem** [08jne01/community-a4e-c#25]
**Describe the bug**
Altough the NW LDG GEAR agrees with the commanded turn, the aircraft can't follow the steering command properly. It seems that the ACFT Nose Wheel skids.

**To Reproduce**
1. Turn on the engine
2. Hold NWS push button
3. Perform sharp turns

**Expected behavior**

**Screenshots**
ACFT MUST turn in the Minimum pavement radius at least.

**Software Information (please complete the following information):**
 - DCS Version: 2.5.6.49798
 - A-4E Version EFM DEV

**Additional context**
I will try to found a general manual that brings the info about the turn radius.

**Comment by malamem** [comment]
According to the manual Chapter GENERAL, the TURN RADIUS(NOSE) IS 20.5.ft

**Comment by 08jne01** [comment]
> According to the manual Chapter GENERAL, the TURN RADIUS(NOSE) IS 20.5.ft

Is this with or without the nosewheel steering engaged?

**Comment by NeroDCS** [comment]
if my interpretation of the natops is correct the NW button only works fpr 45deg hydrolicly , the rest is just coasting freely. see the following picture from the manual for the Nosegear angle vs turn radius
![grafik](https://user-images.githubusercontent.com/46298015/86847380-84043e80-c0ac-11ea-9c59-85e87df18f16.png)

**Comment by 08jne01** [comment]
Yes this is what is currently implemented, (the 3d model animation will not give you a good idea how far it is turned). So we would expect it to turn in a circle with a radius of 21 feet. Sounds about consistent with currently what it is.
